profiles/slurm-sidecar.py

```diff
@@ -0,0 +1,312 @@
+#!/usr/bin/env python3
```

Comment:
This script is autogenerated from the latest cookiecutter for slurm.

Reply:
<OK>